inc/class-antivirus.php

```diff
@@ -579,18 +589,22 @@ public static function show_admin_menu() {
 				'checksum_verifier' => (int) ( ! empty( $_POST['av_checksum_verifier'] ) ),
 			);
 
+			$cron_enabled = $options['cronjob_enable'] || $options['safe_browsing'] || $options['checksum_verifier'];
```

Comment:
Why do not use your new method `_cron_enabled()`?

Reply:
Because we are operating on the non-persistent array `$options` before saving and `_cron_enabled()` relies on the options retrieved by `get_option( 'antivirus' )`

Reply:
+1 Would be much better, one  way to check them. So we haven't not the same checks on several points.

Reply:
Done.